build/ga_build.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash -e
+
+set -o pipefail
+
+find . -type d -name "node_modules" -prune -o -type f -iregex '.*.ts\|.*.js\|.*.tsx\|.*.jsx' -exec ./build/set-mtime-to-md5.sh {} \; && echo Set mtime to MD5 complete!
```

Comment:
I don't recall entirely what this was for - I think it was for another cache we have to reduce build times: https://github.com/DestinyItemManager/DIM/commit/9edecae57e01ec1ef167d107ca833f8e3d8a850a#diff-44575cf5b28512d75644bf54a517dcef304ff809fd511747621b4d64f19aac66

Reply:
Recommend bahmutov/npm-install@v1.4.5 as it work with yarn and handles caching automatically

Reply:
Ok I will check it out, I am trying to get the branch build one happy first.